nobodywho/core/src/chat.rs

```diff
@@ -11,19 +12,22 @@ use llama_cpp_2::model::LlamaModel;
 
 pub struct ChatHandle {
     msg_tx: std::sync::mpsc::Sender<ChatMsg>,
+    pub ready: Arc<AtomicBool>,
```

Comment:
This flag isn't needed after rewriting the FFI using interoptopus. We should get rid of the ready atomicbool.